app/controllers/collections_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CollectionsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @recipes = current_user.collection.recipes
```

Comment:
This was a bit confusing for me. Both here and in the view (below) I would suggest 

```
@collection = current_user.collection
```
...and then iterate through `@collection.recipes` in the view (or a helper...)

Reply:
:+1: